src/hvps/commands/caen/channel.py

```diff
@@ -100,7 +102,9 @@ def _get_set_channel_command(
     if not 0 <= bd <= 31:  # TO DO: change 31 with real number of modules
         raise ValueError(f"Invalid board number '{bd}'. Must be in the range 0..31.")
     if not 0 <= channel <= 8:  # TO DO: change 31 with real number of channels
-        raise ValueError(f"Invalid channel number '{channel}'. Must be in the range 0..31.")
+        raise ValueError(
+            f"Invalid channel number '{channel}'. Must be in the range 0..31."
+        )
```

Comment:
Check if `0..31` range is correct (and remove comment in case it is)
Update / remove comments for channels
Update `ValueError` test with the correct maximum range (0..7)